variables.tf

```diff
@@ -1,130 +1,137 @@
-  default = "10.3.0.0/24"
+  default     = "10.3.0.0/24"
 }
 
-variable "cluster_domain_suffix" {
-  description = "Queries for domains with the suffix will be answered by kube-dns"
-  type        = "string"
-  default     = "cluster.local"
-}
 
 variable "container_images" {
+  type        = map(string)
   description = "Container images to use"
-  type        = "map"
 
   default = {
-    calico           = "quay.io/calico/node:v3.4.0"
-    calico_cni       = "quay.io/calico/cni:v3.4.0"
-    flannel          = "quay.io/coreos/flannel:v0.10.0-amd64"
-    flannel_cni      = "quay.io/coreos/flannel-cni:v0.3.0"
-    kube_router      = "cloudnativelabs/kube-router:v0.2.4"
-    hyperkube        = "k8s.gcr.io/hyperkube:v1.13.2"
-    coredns          = "k8s.gcr.io/coredns:1.3.0"
-    pod_checkpointer = "quay.io/coreos/pod-checkpointer:83e25e5968391b9eb342042c435d1b3eeddb2be1"
+    calico                  = "quay.io/calico/node:v3.13.1"
+    calico_cni              = "quay.io/calico/cni:v3.13.1"
+    coredns                 = "k8s.gcr.io/coredns:1.6.7"
+    flannel                 = "quay.io/coreos/flannel:v0.12.0-amd64"
+    flannel_cni             = "quay.io/coreos/flannel-cni:v0.3.0"
+    kube_apiserver          = "k8s.gcr.io/kube-apiserver:v1.18.1"
+    kube_controller_manager = "k8s.gcr.io/kube-controller-manager:v1.18.1"
+    kube_scheduler          = "k8s.gcr.io/kube-scheduler:v1.18.1"
+    kube_proxy              = "k8s.gcr.io/kube-proxy:v1.18.1"
+    # experimental
+    kube_router = "cloudnativelabs/kube-router:v0.3.2"
   }
 }
 
-variable "enable_reporting" {
-  type        = "string"
-  description = "Enable usage or analytics reporting to upstream component owners (Tigera: Calico)"
-  default     = "false"
-}
 
 variable "trusted_certs_dir" {
+  type        = string
   description = "Path to the directory on cluster nodes where trust TLS certs are kept"
-  type        = "string"
   default     = "/usr/share/ca-certificates"
 }
 
 variable "create_ca" {
+  type        = bool
   description = "Toggles creation of a CA (omit ca_certificate when true)"
   default     = true
 }
 
-variable "ca_certificate" {
-  description = "Existing PEM-encoded CA certificate (generated if blank)"
-  type        = "string"
-  default     = ""
-}
-
-variable "ca_key_alg" {
-  description = "Algorithm used to generate ca_key (required if ca_cert is specified)"
-  type        = "string"
-  default     = "RSA"
+variable "enable_reporting" {
+  type        = bool
+  description = "Enable usage or analytics reporting to upstream component owners (Tigera: Calico)"
+  default     = false
 }
 
-variable "ca_private_key" {
-  description = "Existing Certificate Authority private key (required if ca_certificate is set)"
-  type        = "string"
-  default     = ""
+variable "enable_aggregation" {
+  type        = bool
+  description = "Enable the Kubernetes Aggregation Layer (defaults to false, recommended)"
+  default     = false
 }
 
 variable "apiserver_arguments" {
```

Comment:
Needs to be passed to the apiserver template, since we use this to set OIDC flags

https://github.com/TakeScoop/terraform-render-bootstrap/blob/upstream_master/resources/manifests/kube-apiserver.yaml#L65